### PR TITLE
SAVAMC2-62 Add Joint Pain

### PR DESCRIPTION
**Pull request by ChristineDuong**
# Description
Jira Ticket: SAVAMC2-62

Adds Joint pain to the symptoms list. Renamed "Muscle Pain" to "Muscle and Body Aches". 

Make sure to run `bundle exec rake admin:import_or_update_jurisdictions`

# (Feature) Demo/Screenshots
Insert demo video or photos for a new or updated feature or capability.

# (Bugfix) How to Replicate
Insert steps for how to replicate the bug this PR addresses.

# (Bugfix) Solution
Insert description for the solution this PR implements to address the bug.

# Important Changes
Please list important files (meaning substantial or integral to the PR) along with a list of the general changes that should be highlighted for reviewers.

`example_file.js`
- Example change (ex: refactored import function)

# Testing
This fix was tested on the following browsers (submitter must check all those that apply):
* [x] Chrome
* [ ] Firefox
* [ ] Safari
* [ ] IE11

Please describe the tests needed to verify your changes. Provide instructions for reproducing these tests. List any relevant details for your test configuration.

### CODE REVIEW

File: app/models/patient.rb [hunk 1]
```diff
@@ -146,17 +146,21 @@ def latest_transfer
...
+      .joins(:dosages)
       .where(pause_notifications: false)
       .where('patients.id = patients.responder_id')
       .where.not('latest_assessment_at >= ?', Time.now.in_time_zone('Eastern Time (US & Canada)').beginning_of_day)
       .or(
         where(purged: false)
+          .joins(:dosages)
           .where(pause_notifications: false)
           .where('patients.id = patients.responder_id')
           .where(latest_assessment_at: nil)
```
Comment (jkufro) [inline app/models/patient.rb]: Can we change this to the below? It seems like the whole OR block is just to check if the latest assessment is null when we could just check both in the same where check.
```
scope :reminder_eligible, lambda {
    where(purged: false)
      .joins(:dosages)
      .where(pause_notifications: false)
      .where('patients.id = patients.responder_id')
      .where('last_assessment_reminder_sent IS NULL OR latest_assessment_at <= ?', Time.now.in_time_zone('Eastern Time (US & Canada)').beginning_of_day)
      .distinct
  }
```

**Review by jkufro — CHANGES_REQUESTED** [review]
Just left one comment to change the scope to be more readable

File: app/models/patient.rb [hunk 2]
```diff
@@ -146,17 +146,21 @@ def latest_transfer
 
   # Most recent vaccine dosage
   def latest_dosage
-    dosages.order(created_at: :desc).first
+    # dosages.order(created_at: :desc).first
```
Comment (jkufro) [inline app/models/patient.rb]: Can we remove the commented out line?

File: app/models/patient.rb [hunk 3]
```diff
@@ -656,25 +660,41 @@ def send_assessment(send_now: false)
...
+      # Questionnaires will be sent to Recipient weekly until Dose 2 is administered
+      # Dose 2 is administered between 21 and 28 days after Dose 1 has been administered.
+      # Daily questionnaires sent daily for 7 days after dose 2
+      # Questionnaire sent Weekly for 6 weeks after daily Dose 2 questionnaires completed.
+      # Questionnaires will be sent to Recipient once at the 6 and 12 months marks.
+      case latest_dosage&.dose_number
+      when 1
+        return unless (dose_date > 7.days.ago.to_date) || (difference % 7 == 0)
+      when 2
+        return unless (dose_date > 7.days.ago.to_date) || (difference % 7 == 0 && difference / 7 < 6 && difference / 7 > 0 ) || ((difference % 30 == 0 && (difference / 30 == 6 || difference / 30 == 12 ))
```
Comment (jkufro) [inline app/models/patient.rb]: Would it make more sense to use `(Time.now.getlocal(address_timezone_offset) - 7.days).to_date`

File: app/models/patient.rb (hunk 1 above)
Comment (ChristineDuong) [reply app/models/patient.rb]: Changed in 93ea627

File: app/models/patient.rb (hunk 2 above)
Comment (ChristineDuong) [reply app/models/patient.rb]: Removed in 93ea627

**Review by kevstoy — APPROVED** [review]

**Review by kevstoy — APPROVED** [review]

**Review by jkufro — APPROVED** [review]
approved updates

**Pull request merged**